src/test/javascript/portal/data/ServerSpec.js

```diff
@@ -26,7 +39,15 @@ describe('Portal.data.Server', function() {
             expect(Portal.data.Server.getInfo('http://server2/someExtraPath')).toBe(server2Info);
         });
 
-        it('returns undefined for unknown server', function() {
+        it('not blocked by protocol ', function() {
+            expect(Portal.data.Server.getInfo('http://maybesecureserver3')).not.toBe(Portal.data.Server.UNKNOWN);
+        });
+
+        it('is blocked', function() {
+            expect(Portal.data.Server.getInfo('http://rogueServer')).toBe(Portal.data.Server.UNKNOWN);
+        });
+
+        it('returns server regardless of HTTP protocol', function() {
```

Comment:
Not sure we're testing this here.  The result is exactly the same as for the above two test cases.

Reply:
Yes agreed. Ive removed the duplication now.